package.json

```diff
@@ -19,8 +19,7 @@
     "builder:check": "eslint config",
     "test-func-remote": "TEST_FUNC=remote mocha --opts ./test/func/mocha.opts",
     "test-func-static": "TEST_FUNC=static mocha --opts ./test/func/mocha.opts",
-    "test-func-dev": "TEST_FUNC=dev mocha --opts ./test/func/mocha.opts",
-    "install-selenium": "selenium-standalone install"
```

Comment:
You don't always need this. Just when running tests on same machine (local or local Travis / not sauce). So perhaps best as a first class script command and then have root project call it in a dev doc and in Travis CI, not postinstall (eg always)